mjpeg/__init__.py

```diff
@@ -66,6 +71,11 @@ def skip_data(stream, left):
         left -= len(rv)
 
 
+async def askip_data(stream: aiohttp.BodyPartReader, left: int):
+    buf = bytearray(left)
+    await aread_data(buf, stream, left)
+
+
```

Comment:
Is it necessary to create a byte array and copy data into it here? The function askip_data discards the byte array immediately. Would it be possible to simply read the data from the stream without copying it into another buffer?

MJPEG stream chunks can be quite big and I would like to try to minimize the overhead here.